ks-prometheus/manifests/whizard-telemetry-prometheusRule.yaml

```diff
@@ -111,7 +117,39 @@ spec:
     - expr: |
         sum by (node, cluster)(node_filesystem_files{job="node-exporter", device=~"/dev/.*",device!~"/dev/loop\\d+"} - node_filesystem_files_free{job="node-exporter", device=~"/dev/.*",device!~"/dev/loop\\d+"}) *  on(node, cluster) group_left(host_ip, role) max by(node, host_ip, role, cluster) (workspace_workload_node:kube_pod_info:{node!="",host_ip!=""})
       record: node:node_inodes_used_total:sum
-  - name: whizard-telemetry-apiserver-recording.rules
+  - name: whizard-telemetry-namespace.rules
+    rules:
+    - expr: |
+        sum by(cluster,namespace,workload,workload_type)(node_namespace_pod_container:container_cpu_usage_seconds_total:sum_irate{} * on(cluster,pod)group_left(workload,workload_type) workspace_workload_node:kube_pod_info:{})
```

Comment:
`on(cluster,pod)` -> `on(cluster,namespace,pod)`
Other similar rules may be to change too